# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/prim_store/interned.rs
# Commit: c912923296d1
# Full Hash: c912923296d1fba93b6681dd44a3902d7689ef02
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-25 03:42:54
# Regressor Bug: 1769963
# File Overlap Count: 1
# Description:
#   Bug 1769963 - Fix backdrop-filter inside complex transform with CSS filter chain r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147124
# ==============================================================================

diff -r dd7e11f8b419 -r c912923296d1 gfx/wr/webrender/src/prim_store/interned.rs
--- a/gfx/wr/webrender/src/prim_store/interned.rs	Tue May 24 22:14:40 2022 +0000
+++ b/gfx/wr/webrender/src/prim_store/interned.rs	Tue May 24 22:53:15 2022 +0000
@@ -4,7 +4,7 @@
 
 // list of all interned primitives to match enumerate_interners!
 
-pub use crate::prim_store::backdrop::Backdrop;
+pub use crate::prim_store::backdrop::{BackdropCapture, BackdropRender};
 pub use crate::prim_store::borders::{ImageBorder, NormalBorderPrim};
 pub use crate::prim_store::image::{Image, YuvImage};
 pub use crate::prim_store::line_dec::{LineDecoration};